README_building.md

```diff
@@ -94,10 +94,23 @@ With Cmake directly:
 
 Cmake will automatically create the `./build` folder if it does not exist, and it wil configure the project.
 
-Instead, if you have CMake version 3.21+, you can use one of the configuration presets that are listed in the CmakePresets.json file.
+If you have CMake version 3.21+ installed on your system, you can use one of the configuration presets instead.
```

Comment:
@ClausKlein Are those changes in the documentation more descriptive or I missed something?

Reply:
If the user has a `multiconfig `project created, than the user may switch with `--config` i.e. from `Debug` to `Release`.
Is this your intention? If yes, it should be documented. And note the differerent argument with` ctest --build-config` 

````bash
cmake . --preset=ninja-multi
cmake --build --preset=ninja-multi --config Release
cmake --build --preset=ninja-multi --config Release --target help # or install ...
ctest --preset=ninja-multi   --build-config Release
````

And perheps, show your preset list, the output of `cmake . --list-presets=all`

Reply:
Hmmm the configuration setting is embedded in the build presets this is why the names for build presets have a `-debug` or `-release` suffix. This is needed to have presets that work with IDEs. However, I cannot forbid a user to pass a `--config` argument in the cli and if I am not mistaken this will override the preset setting. So I would prefer the users to use the build presets "as is".